onchain/rollups/test/foundry/inputs/InputBox.t.sol

```diff
@@ -167,6 +167,26 @@ contract InputBoxTest is Test {
         assertEq(inputBox.getNumberOfInputs(_dapp), 0);
     }
 
+    function newInput(
+        uint256 _len
+    ) internal pure returns (bytes memory input_) {
+        assembly {
+            let ptr := mload(0x40)
+            input_ := ptr
+            mstore(input_, _len)
+            mstore(0x40, add(ptr, _len))
+        }
+    }
+
+    function testAddLargeInput() public {
+        address dapp = vm.addr(1);
+
+        inputBox.addInput(dapp, newInput(CanonicalMachine.INPUT_MAX_SIZE));
```

Comment:
do you think we can simplify the code to 
`inputBox.addInput(dapp, new bytes(CanonicalMachine.INPUT_MAX_SIZE));`
and remove the function `newInput`

Reply:
Oh, I didn't know you could create a new byte array that way! Thanks, this will surely simplify the code.